content/en/docs/reference/command-line-tools-reference/feature-gates/cbor-serving-and-storage.md

```diff
@@ -0,0 +1,15 @@
+---
+title: CBORServingAndStorage
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.32"
+---
+Enables CBOR as a [supported encoding for requests and
+responses](/docs/reference/using-api/api-concepts/#cbor-encoding), and as the preferred storage
+encoding for custom resources.
```

Comment:
:thought_balloon: if you enable this feature gate, and write a custom resource, and then you disable the feature gate, does that trigger the need for special deletion (KEP 3926 style).

Even though it's alpha, I wonder if we're introducing a new way for people to break their clusters. If so, how about a post-release blog article that runs people through the unbreaking?

Reply:
For beta, we should document that detail. Not a concern for alpha but it's still good to document.

Also good to mention if we blog about this.